pkg/client/network.go

```diff
@@ -6,17 +6,28 @@ SPDX-License-Identifier: Apache-2.0
 
 package client
 
+import proto "github.com/hyperledger/fabric-gateway/protos"
+
 // Network represents a blockchain network, or Fabric channel. The Network can be used to access deployed smart
 // contracts, and to listen for events emitted when blocks are committed to the ledger.
 type Network struct {
-	gateway *Gateway
-	name    string
+	client    proto.GatewayClient
+	signingID *signingIdentity
+	name      string
+}
+
+// GetDefaultContract returns a Contract representing the default smart contract for the named chaincode.
+func (network *Network) GetDefaultContract(chaincodeID string) *Contract {
+	return network.GetContract(chaincodeID, "")
 }
 
-// GetContract returns a Contract representing the named smart contract.
-func (network *Network) GetContract(name string) *Contract {
+// GetContract returns a Contract representing a smart contract within a named chaincode.
+func (network *Network) GetContract(chaincodeID string, contractName string) *Contract {
```

Comment:
Could we use the existing GoSDK method names for these to help migrating applications?
```
func (n *Network) GetContract(chaincodeID string) *Contract
func (n *Network) GetContractWithName(chaincodeID string, name string) *Contract
```
https://godoc.org/github.com/hyperledger/fabric-sdk-go/pkg/gateway

Reply:
Done.